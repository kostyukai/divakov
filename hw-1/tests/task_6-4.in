6865.0006
