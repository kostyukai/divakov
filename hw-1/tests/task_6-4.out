0100000010111010110100010000000000100111010100100101010001100001
